erna/scripts/submit_runlist.py

```diff
@@ -0,0 +1,65 @@
+import click
+import pandas as pd
+
+from ..automatic_processing.database_utils import insert_new_jobs
+from ..utils import load_config
+from datetime import date
+
+from ..automatic_processing.database import (
+    database,
+    RawDataFile,
+    Jar,
+    XML,
+    Queue,
+    Job,
+)
+
+
+@click.command()
+@click.argument('runlist')
+@click.argument('jar')
```

Comment:
maybe call it jar_version to make it clear what it is (Camelcase dosn't work with click by the way)

Reply:
It's called jar everywhere in erna. If we want to change this, not in this PR